src/types.ts

```diff
@@ -36,6 +36,7 @@ export interface ITransformEndpointOperationOpts<T extends Fragment> {
   document: T;
   name: string;
   method: string;
+  key?: string;
```

Comment:
Had to add it because `transformOas3Operation` [uses](https://github.com/stoplightio/http-spec/pull/262/files#diff-edf0102d749d7bcfcaf6f168492e0d23f08a03740e99332863026068cccc3feeR47) this type